modules/twinklespeedy.js

```diff
@@ -41,21 +41,24 @@ Twinkle.speedy.dialog = null;
 // The speedy criteria list can be in one of several modes
 Twinkle.speedy.mode = {
 	sysopSubmit: 1,  // radio buttons, no subgroups, submit when "Submit" button is clicked
-	sysopRadioClick: 2,  // radio buttons, no subgroups, submit when a radio button is clicked
-	userMultipleSubmit: 3,  // check boxes, subgroups, "Submit" button already pressent
-	userMultipleRadioClick: 4,  // check boxes, subgroups, need to add a "Submit" button
-	userSingleSubmit: 5,  // radio buttons, subgroups, submit when "Submit" button is clicked
-	userSingleRadioClick: 6,  // radio buttons, subgroups, submit when a radio button is clicked
+	sysopMultipleSubmit: 2, // check boxes, subgroups, "Submit" button already present
```

Comment:
The mode constants were very carefully laid out in an odd-even pattern to satisfy the code at line 247. Please test the code again with the "When to go ahead and tag/delete the page" Twinkle preference set to "As soon as I click an option" (i.e. speedySelectionStyle = radioClick) and fix the code as required. 

I also wonder whether there need to be two sysop modes. The reason why tagging has separate single and multiple modes is that some CSD templates (e.g. db-hoax and the A7 subcategories) are not properly supported by db-multiple, so we have to prevent a different list to the user. (Arguably they could be combined into one, but that's a separate issue for another day.) For actual page deletion, is there any reason why we couldn't just get rid of the radio-button list and *always* show checkboxes?